src/test/resources/dlsfls/securityconfig_tlq.yml

```diff
@@ -0,0 +1,17 @@
+---
+_meta:
+  type: "config"
+  config_version: 2
+config:
+  dynamic:
+    do_not_fail_on_forbidden: true
```

Comment:
Why `do_not_fail_on_forbidden` needs to be `true`?

Reply:
This is a precondition for the TLQ to work.